src/runtime/deserializer.ts

```diff
@@ -56,7 +56,7 @@ export function deserialize(
         (o, k) => k === null ? o : o[k],
         v,
       );
-      parent[refPath.at(-1)!] = target;
+      parent[refPath.slice(-1)[0]!] = target;
```

Comment:
Cloning the array just to get the last value seems a bit much. Let's skip the temporary array allocation here by accessing the last item directly.

```suggestion
      parent[refPath[refParent.length - 1]!] = target;
```

Reply:
Oh, sure! Thank you =D